integrated_channels/cornerstone/constants.py

```diff
@@ -0,0 +1,5 @@
+"""
+Constants used by the Cornerstone integrated channel
+"""
+
+ISO_8601_DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
Good work but you should put this in parent module (integrated channel) so in future other channels can also use this.

Reply:
Done.